docs/static-pods.md

```diff
@@ -0,0 +1,100 @@
+
+For example, this is how to start a simple web server as a static pod:
+
+1. Choose a node where we want to run the static pod. In this example, it's `my-minion1`.
+
+        $ ssh my-minion1
+
+2. Choose a directory, say `/etc/kubelet.d` and place a web server pod definition there, e.g. `/etc/kubernetes.d/static-web.yaml`:
+
+        my-minion1$ mkdir /etc/kubernetes.d/
+        my-minion1$ cat <<EOF >/etc/kubernetes.d/static-web.yaml
+        apiVersion: v1beta3
+        kind: Pod
+        metadata:
+          name: static-web
+          labels:
+            role: myrole
+        spec:
+          containers:
+            - name: web
+              image: nginx
+              ports:
+                - name: web
+                  containerPort: 80
+                  protocol: tcp
+        EOF
+
+2. Configure your kubelet daemon on the node to use this directory by running it with `--config=/etc/kubelet.d/` argument.  On Fedora Fedora 21 with Kubernetes 0.17 edit my `/etc/kubernetes/kubelet` to include this line:
+        KUBELET_ARGS="--cluster_dns=10.254.0.10 --cluster_domain=kube.local --config=/etc/kubelet.d/"
+
+     Instructions for other distributions or Kubernetes installations may vary.
+
+3. Restart kubelet. On Fedora 21, this is:
+
+        my-minion1$ systemctl restart kubelet
+
+## Pods created via HTTP
+Kubelet periodically downloads a file specified by `--manifest-url=<URL>` argument and interprets it as a json/yaml file with a pod definition. It works the same as `--config=<directory>`, i.e. it's reloaded every now and then and changes are applied to running static pods (see below).
+
+## Behavior of static pods
+When kubelet starts, it automatically starts all pods defined in directory specified in `--config=` or `--manifest-url=` arguments, i.e. our static-web.  (It may take some time to pull nginx image, be patient…):
+
+    my-minion1$ docker ps
+    CONTAINER ID IMAGE         COMMAND  CREATED        STATUS              NAMES
+    f6d05272b57e nginx:latest  "nginx"  8 minutes ago  Up 8 minutes        k8s_web.6f802af4_static-web-fk-minion1_default_67e24ed9466ba55986d120c867395f3c_378e5f3c
+
+If we look at our Kubernetes API server (running on host `my-master`), we see that a new mirror-pod was created there too:
+
+    $ ssh my-master
+    my-master$ kubectl get pods
+    POD                     IP           CONTAINER(S)   IMAGE(S)    HOST                        LABELS       STATUS    CREATED         MESSAGE
+    static-web-my-minion1   172.17.0.3                              my-minion1/192.168.100.71   role=myrole  Running   11 minutes
+                                         web            nginx                                                Running   11 minutes
+
+Labels from the static pod are propagated into the mirror-pod and can be used as usual for filtering.
+
+Notice we cannot delete the pod with the API server (e.g. via `kubeclt` command), kubelet simply won't remove it.
```

Comment:
typo: kubectl

Should also link to the doc. Use relative link (from getting-started): ../user-guide/kubectl